spec/system/admin/editing_an_estimate_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe(
+  "Editing an estimate", :js,
+  feature_setting: { user_based_price_groups: true },
+) do
+  let(:facility) { create(:setup_facility) }
+  let(:director) { create(:user, :facility_director, facility:) }
+  let!(:user) { create(:user) }
+  let(:price_group) { user.price_groups.first }
+  let!(:item) { create(:setup_item, facility:) }
+  let!(:item_price_policy) { create(:item_price_policy, product: item, price_group:) }
+  let!(:other_item) { create(:setup_item, facility:) }
+  let!(:other_item_price_policy) { create(:item_price_policy, product: other_item, price_group: price_group) }
+  let!(:instrument) { create(:setup_instrument, facility:) }
+  let!(:instrument_price_policy) { create(:instrument_price_policy, product: instrument, price_group:) }
+
+  let!(:estimate) do
```

Comment:
Is there any reason to do this instead of having another `let` for the `estimate_details`?